src/components/MiniToc/MiniToc.tsx

```diff
@@ -47,8 +47,32 @@ const MiniToc = memo<MinitocProps>(({headings, router, headerHeight, onItemClick
         return null;
     }
 
+    const miniTocContent = flatHeadings.map(({href, title, isChild}, index) => {
+        if (index === 0) {
+            return (
+                <li
+                    key={href}
+                    data-hash={href}
+                    className={b('section', {child: isChild})}
+                    aria-current="location"
+                >
+                    <a href={href} className={b('section-link')} data-router-shallow>
+                        {title}
+                    </a>
+                </li>
+            );
+        }
+        return (
+            <li key={href} data-hash={href} className={b('section', {child: isChild})}>
+                <a href={href} className={b('section-link')} data-router-shallow>
+                    {title}
+                </a>
+            </li>
+        );
+    });
+
     return (
-        <div className={b()}>
+        <div className={b()} aria-label={t('article navigation')}>
```

Comment:
label should be `article-navigation` and it need to be aded here and in "ru" 
https://github.com/diplodoc-platform/components/blob/master/src/i18n/en.json#L50